packages/rocketchat-livechat/assets/rocket-livechat.js

```diff
@@ -716,6 +716,9 @@
 
 	// exports
 	w.RocketChat.livechat = {
+		// objects
+		api	: api,
```

Comment:
the `api` was intended to be the communication channel between rocket.chat instance and the livechat widget and not to be opened to the anyone.